README.md

```diff
@@ -150,9 +150,20 @@ This can be done by using option like so: `'lookup-request.format.customFormatNa
 In this case, the `fail-on-missing-field` will be passed to `SerializationFormatFactory::createEncodingFormat(
 DynamicTableFactory.Context context, ReadableConfig formatOptions)` method in `ReadableConfig` object.
 
-In configuration when default, Flink-Json format is used for `GenericGetQueryCreator`, all options defined in [json-format](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/formats/json/)
+In configuration with default, Flink-Json format is used for `GenericGetQueryCreator`, all options defined in [json-format](https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/formats/json/)
 can be passed through table DDL. For example `'lookup-request.format.json.fail-on-missing-field' = 'true'`. In this case, format identifier is `json`.
 
+#### Timeouts
+Lookup Source is guarded by two timeout timers. First one is specified by Flink's AsyncIO operator that executes `AsyncTableFunction`.
+The default value of this timer is set to 3 minutes and can be changed via `table.exec.async-lookup.timeout` [option](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/config/#table-exec-async-lookup-timeout).
+
+The second one is set per individual HTTP requests by HTTP client. It's default value is set currently to 30 seconds and can be chane via `gid.connector.http.source.lookup.request.timeout` option. 
+
+Flink's current implementation of `AsyncTableFunction` does not allow specifying custom logic for handling Flink AsyncIO timeouts as it is for Java API.
+Because of that, if AsyncIO timer pass, Flink will throw TimeoutException which will case job restart.
+
+The HTTP request timeouts on the other hand will not cause Job restart. In that case, proper log information will be printed application logs only.
```

Comment:
Some preposition is missing in this sentence `log information will be printed application logs only.`

Reply:
fixed